public/.htaccess

```diff
@@ -2,21 +2,17 @@
 
 # Deny access to templates (but allow from localhost)
 <Files *.ss>
-    Order deny,allow
-    Deny from all
-    Allow from 127.0.0.1
+    Require ip 127.0.0.1
```

Comment:
Do we actually know what the point of allowing people to see .ss files on localhost is?

Is this just a bit of legacy from logic from before we had the public web-root?

Reply:
Probably. But I went like for like because maybe it'd be a breaking change if I didn't.